app/src/main/java/org/mozilla/fenix/compose/Favicon.kt

```diff
@@ -23,42 +22,49 @@ import mozilla.components.browser.icons.IconRequest
 import mozilla.components.browser.icons.compose.Loader
 import mozilla.components.browser.icons.compose.Placeholder
 import mozilla.components.browser.icons.compose.WithIcon
-import mozilla.components.ui.colors.PhotonColors
 import org.mozilla.fenix.components.components
 import org.mozilla.fenix.theme.FirefoxTheme
 import org.mozilla.fenix.theme.Theme
 
 /**
  * Load and display the favicon of a particular website.
  *
- * @param url Website [URL] for which the favicon will be shown.
+ * @param url Website [url] for which the favicon will be shown.
  * @param size [Dp] height and width of the image to be loaded.
+ * @param modifier [Modifier] allowing to control among others the dimensions and shape of the image.
```

Comment:
```suggestion
 * @param modifier [Modifier] to be applied to the layout.
```
Don't want too many original description of modifier. So just reusing the one that's already in the codebase and aligns with Jetpack Compose docs.

<img width="703" alt="Screen Shot 2022-04-19 at 8 30 59 PM" src="https://user-images.githubusercontent.com/1190888/164122732-58d535f8-f351-4296-ac49-6894cde742bd.png">